### PR TITLE
Add text when the card is not found (#3005)

### PR DESCRIPTION
**Pull request by olivierPigeon-RTE**
Signed-off-by: olivierPigeon-RTE <olivier.pigeon@rte-france.com>

- In release note :
  -  In chapter :  Tasks
  -  Text : #3005 :  Add text when trying to open a card that is not found in the feed

### CODE REVIEW

File: ui/main/src/app/modules/cards/components/card-details/card-details.component.html [hunk 1]
```diff
@@ -24,4 +25,6 @@
 
   <br/>
   <br/>
-</div>
\ No newline at end of file
+</div>
+
+<div *ngIf="cardNotFound" id="opfab-feed-card-not-found" style="font-size:3em; text-align:center;" translate>feed.cardNotFound</div>
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/cards/components/card-details/card-details.component.html:30]: Smaller font and text should be more down (it is too close to the timeline)

File: src/test/cypress/cypress/integration/Translation.spec.js [hunk 2]
```diff
@@ -351,6 +334,21 @@ describe('Test translations', function () {
         checkNotificationSeverityTexts('Alarm', 'Actie', 'Conform', 'Informatie');
         checkAknowledgementTexts('Bevestigen', 'Alles', 'Bevestigd', 'Niet bevestigd');
         checkDateFilterTexts('Ontvangstdatum', 'START', 'EIND', 'Filters toepassen op tijdlijn', 'Opnieuw installen');
-    })
+    });
 
-})
\ No newline at end of file
+    it('Check Feed translations', function () {
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/Translation.spec.js:339]: it('Check translation for non-existent card' ...

File: src/test/cypress/cypress/integration/FeedTests.spec.js [hunk 3]
```diff
@@ -61,15 +64,16 @@ describe ('FeedScreen tests',function () {
...
             .as('firstCardUrlId')
             .then((urlId) => {
-                cy.hash().should('eq', '#/feed/cards/'+urlId);
+                cy.hash().should('eq', '#/feed/cards/' + urlId);
                 cy.get('of-card-details').find('of-detail');
             });
+        cy.get('#opfab-feed-card-not-found').should('not.exist');
 
-
+        tryToLoadNonExistingCard();
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/FeedTests.spec.js:76]: I think it is not necessary to have this test as it's redundant with tests in translation.spec.ts

Comment (olivierPigeon-RTE) [reply src/test/cypress/cypress/integration/FeedTests.spec.js:76]: I did 2 separate tests in 2 different places that have different goals : 
* the one in FeedTest tests the card that was displayed before trying to load a non existing card is not displayed anymore and is replaced by a text (the value of the text is not tested here). And the same way, when we load an existing card, the texts disappears and the real card is shown
* the test in Translation chacks the value of the string in the message

I think these are 2 very different things, this is why I did separate tests in separate files

Comment (freddidierRTE) [reply src/test/cypress/cypress/integration/FeedTests.spec.js:76]: ok

**Pull request merged**